hera_pspec/uvpspec.py

```diff
@@ -32,7 +33,7 @@ def __init__(self):
         self._Nfreqs = PSpecParam("Nfreqs", description="Number of unique frequency bins in the data.", expected_type=int)
         self._Npols = PSpecParam("Npols", description="Number of polarizations in the data.", expected_type=int)
         self._history = PSpecParam("history", description='The file history.', expected_type=str)
-
+        self._r_params = PSpecParam("r_params", description = 'r_params.', expected_type = str)
```

Comment:
Important to have a clearer description here I think!

Reply:
I've improved this description.